chain/chain/src/chain.rs

```diff
@@ -391,6 +396,58 @@ pub fn check_known(
     check_known_store(chain, block_hash)
 }
 
+pub static CRYPTO_HASH_TIMER_RESULTS: Lazy<Mutex<LruCache<CryptoHash, u64>>> =
+    Lazy::new(|| Mutex::new(LruCache::new(10000)));
```

Comment:
Probably deserves some comments here on what crypto hashes and times you are putting here, since this is a global variable.

Reply:
good idea - done